subjects/java/checkpoints/date-formatter/README.md

```diff
@@ -0,0 +1,72 @@
+## Date Formatter
+
+### Instructions
+
+Write a class `DateFormatter` that has three private attributes:
+
+- `date`: the date to be formatted (as a date).
+- `format`: the format to convert the date to.
+- `formattedDate`: contains the date converted into the given format.
+
+Create getters for all the attributes and setters for `date` and `format`. The conversion should happen when you set the `format` or the `date`. If the `format` is not correct, the conversion shouldn't happen. Additionally, create methods to output the date in the specified `format`, and constructors with only the `date`, with `date` and `format`, and a default constructor.
```

Comment:
The default format should be specified somewhere

Reply:
in the constructor